applications/data-captor/Readme.md

```diff
@@ -0,0 +1,21 @@
+#data-captor is the vision captor helm chart with data-captor
+
```

Comment:
There is no proper details about pre-requisites and details about OpenNESS or Smart edge Open version used for this application?